Edits to PGD 5.5 release (#5610)
EnterpriseDB/docs: product_docs/docs/pgd/5/quickstart/quick_start_docker.mdx

You also ask TPA to generate repeatable hostnames for the nodes by passing `--hostnames-unsorted`. Otherwise, it selects hostnames at random from a predefined list of suitable words.

> Finally, `--keyring-backend legacy` tells that the keyring backend is the legacy backend. Secrets are stored with an older keyring backend, as the version of Ubuntu this example is based on doesn't support the newer keyring backend.

Reviewer: ```suggestion
Finally, `--keyring-backend legacy` tells TPA to select the legacy version of the keyring backend. Secrets are stored with an older keyring backend, as the version of Ubuntu this example is based on doesn't support the newer keyring backend.
```

### Deploying the cluster

> You can now [deploy](/tpa/latest/tpaexec-deploy/) the distributed cluster. The `tpaexec` command runs its `provision` command to prepare the Docker containers for the deployment:

Reviewer: ```suggestion
You can now [deploy](/tpa/latest/tpaexec-deploy/) the distributed cluster. 
For Docker deployments, this will both provision the required Docker containers and deploy the software to those containers:
```
